plopfile.mjs

```diff
@@ -0,0 +1,150 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+
+const vectaraWebsiteQuestions = require("./config/website-vectara/queries.json");
+const vectaraDocsQuestions = require("./config/vectara-docs/queries.json");
+const askFeynmanQuestions = require("./config/ask-feynman/queries.json");
+
+const DEFAULT_CONFIGS = {
+  "vectara-docs": {
```

Comment:
Can we configure `app_title` for each of these so it's clear which app is running when you look at it in the browser?

For example I'd expect this title to be "Ask Feynman".

![image](https://github.com/vectara/vectara-answer/assets/1238659/e53003c7-acbb-4f44-a261-ff0c5b034e93)

Reply:
https://github.com/vectara/vectara-answer/pull/99/commits/25ddc03a88a276af279e5b0b277f33abb0395eec